### PR TITLE
Change integration tests dialect

### PR DESCRIPTION
**Pull request by westito**
I changed integration tests' dialect to postgres to make compatible with it. It is compatible with sqlite too.

### CODE REVIEW

**Review by simolus3 — COMMENTED** [review]
Thanks for fixing the tests (and the `logStatements` parameter, nice call!).

Still, I think that the examples intended to be used by Flutter developers should still be having `sqlite` as their selected dialect. First, the detailed options like the sqlite3 version and enabled modules only really make sense with that dialect. Also, Flutter users are almost certainly using sqlite as their DBMS, so keeping that in the examples seems like the best call.

**Comment by westito** [comment]
Yes, you're right. I was too diligent 😄

File: tool/test_all.sh [hunk 1]
```diff
@@ -21,6 +22,7 @@ function run_test() {
 function run_test_flutter() {
     echo "------------- Running flutter test: $1 -------------"
     pushd $1 > /dev/null
+    rm -rf .dart_tool
```
Comment (simolus3) [inline tool/test_all.sh:25]: Are those necessary? Doing `pub upgrade` and a rebuild should have the same effects as clearing that folder, right? Does this fix anything that I'm overlooking?

Comment (westito) [reply tool/test_all.sh:25]: Maybe not. It is just my "just for sure" habbit. I can remove this line if you think so

Comment (simolus3) [reply tool/test_all.sh:25]: Let's remove them, that way rebuilds may be a bit faster. If it turns out to be a problem we can bring them back later.

Comment (westito) [reply tool/test_all.sh]: Okay. I removed

**Review by simolus3 — APPROVED** [review]
Thanks!

**Pull request merged**